kolibri/plugins/setup_wizard/api_urls.py

```diff
@@ -1,15 +1,15 @@
-from django.conf.urls import include
-from django.conf.urls import url
 from rest_framework import routers
 
 from .api import FacilityImportViewSet
+from .api import SetupWizardResource
 from .api import SetupWizardRestartZeroconf
 
-router = routers.DefaultRouter()
+router = routers.SimpleRouter()
```

Comment:
is this a Vue upgrade, or a change you decided on? I don't have any context on what the difference, if any, is

Reply:
I think @rtibbles might be better able to explain the difference -- but if I didn't do this, something broke and when I changed it things worked :)

Reply:
The DefaultRouter does some introspection to generate an API view at the root of where the router is mounted: https://www.django-rest-framework.org/api-guide/routers/#defaultrouter

My assumption is that this introspection probably breaks because of our own tweaks and adjustments to DRF.